### PR TITLE
Fix navigation editor color materials not having unique names

### LINKED ISSUE #26024: Navigation Color Editor Settings Do not Work
**Godot version:**
3.1 beta4

**OS/device including version:**
Windows 7 Pro 64bit, GTX 1060 6GB, AMD Ryzen 5 1400

**Issue description:**
The editor's Navigation Edge/Solid/Disabled settings appear to have no or very little effect, rendering them nearly white. This makes it difficult to see in bright scenes.
image

### PR DESCRIPTION
**Pull request by YeldhamDev**
Fixes #26024.

### CODE REVIEW

File: editor/spatial_editor_gizmos.cpp [hunk 1]
```diff
@@ -3584,10 +3584,10 @@ void CollisionPolygonSpatialGizmoPlugin::redraw(EditorSpatialGizmo *p_gizmo) {
 ////
 
 NavigationMeshSpatialGizmoPlugin::NavigationMeshSpatialGizmoPlugin() {
-	create_material("navigation_material", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/navigation_edge", Color(0.5, 1, 1)));
-	create_material("navigation_material", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/navigation_edge_disabled", Color(0.7, 0.7, 0.7)));
-	create_material("navigation_material", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/navigation_solid", Color(0.5, 1, 1, 0.4)));
-	create_material("navigation_material", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/navigation_solid_disabled", Color(0.7, 0.7, 0.7, 0.4)));
+	create_material("edge_material", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/navigation_edge", Color(0.5, 1, 1)));
+	create_material("edge_material_disabled", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/navigation_edge_disabled", Color(0.7, 0.7, 0.7)));
+	create_material("solid_material", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/navigation_solid", Color(0.5, 1, 1, 0.4)));
```
Comment (akien-mga) [inline editor/spatial_editor_gizmos.cpp]: I think those names are global, so it should still start with `navigation_`.

Comment (akien-mga) [reply editor/spatial_editor_gizmos.cpp]: See e.g.:
```
$ rg solid_material
editor/spatial_editor_gizmos.cpp
2176:   create_material("visibility_notifier_solid_material", gizmo_color);
2312:           Ref<Material> solid_material = get_material("visibility_notifier_solid_material", p_gizmo);
2313:           p_gizmo->add_solid_box(solid_material, aabb.get_size(), aabb.get_position() + aabb.get_size() / 2.0);
2325:   create_material("particles_solid_material", gizmo_color);
2465:           Ref<Material> solid_material = get_material("particles_solid_material", p_gizmo);
2466:           p_gizmo->add_solid_box(solid_material, aabb.get_size(), aabb.get_position() + aabb.get_size() / 2.0);
2483:   create_material("reflection_probe_solid_material", gizmo_color);
2642:           Ref<Material> solid_material = get_material("reflection_probe_solid_material", p_gizmo);
2643:           p_gizmo->add_solid_box(solid_material, probe->get_extents() * 2.0);
2659:   create_material("gi_probe_solid_material", gizmo_color);
2816:           Ref<Material> solid_material = get_material("gi_probe_solid_material", p_gizmo);
2817:           p_gizmo->add_solid_box(solid_material, aabb.get_size());
3607:   Ref<Material> solid_material = get_material("navigation_material", p_gizmo);
3608:   Ref<Material> solid_material_disabled = get_material("navigation_material", p_gizmo);
3689:   m->surface_set_material(0, navmesh->is_enabled() ? solid_material : solid_material_disabled);
```

Comment (YeldhamDev) [reply editor/spatial_editor_gizmos.cpp]: Updated.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**